Makefile

```diff
@@ -7,7 +7,7 @@ BRANCH_TO_DELETE :=
 CURRENT_DIRECTORY := $(shell pwd)
 DEPLOYCONFIG ?=
 DEPLOY_TARGET ?=
-GIT_BRANCH ?= $(shell git rev-parse --symbolic-full-name --abbrev-ref HEAD)
+GIT_BRANCH := $(shell if [ -f '.venv/deployed-git-branch' ]; then cat .venv/deployed-git-branch 2> /dev/null; else git rev-parse --symbolic-full-name --abbrev-ref HEAD; fi)
```

Comment:
This will remove the abbility to overwrite GIT_BRANCH on the cmdline. The README.md should reflect this (there's an example there).